mage/how-to-guides/run-mage-modules-on-subgraphs.md

```diff
@@ -0,0 +1,139 @@
+---
+id: run-a-subgraph-module
+title: How to run a MAGE module on subgraphs
+sidebar_label: Run modules on subgraphs
+---
+
+If you have already read our how-to guide on [how to run MAGE modules and graph algorithms](/mage/how-to-guides/run-a-query-module.md),
+now it's time to move on to the finer execution of graph analytics.
+In this how-to guide, we will project a portion of the graph from the whole network persisted in 
+Memgraph, and run our algorithms on that portion of the graph.
```

Comment:
```suggestion
The following how-to guide will demonstrate how to run graph analytics on sub-graphs. A portion of the graph is projected from the whole network persisted in Memgraph, and algorithms are run on that portion of the graph.

If you need help with running MAGE modules and graph algorithms, check out the how-to guide on that topic. 
```

This is documentation, not blog post + it's not a tutorial - so keep the tone like it's an encyclopedia

Reply:
Will do.